### ISSUE TITLE
Will you be making exporters for original .v3d models?

### ISSUE THREAD

**Issue by Nitestokeh** [rafalh/rf-reversed#1]
I was always curious about extracted character and weapon models from RF. You'd be doing God's work if you're able to write a sort of .v3c converter with animations as well

**Comment by rafalh** [comment]
I already created a converter that allows importing meshes to RF (https://github.com/rafalh/rf-tools/tree/master/v3d-tool) If I understand correctly you would like to see conversion in the opposite direction. I'm not planning to implement it anytime soon and I don't see much benefit in doing so. RF models are very low-poly and nowadays it's much better to redo them from scratch instead of importing them from the game.
Technically speaking doing such a converter shouldn't be hard because v3c (animated mesh) and rfa (animations) formats are nearly completely reverse engineered.

**Comment by Nitestokeh** [comment]
RF models have a certain early 3D graphics charm to them and I'd like to convert them to more capable engines for playing around with. I could actually pay for a working extractor.

**Comment by Bartalon** [comment]
I agree with Nitestokeh about following through with a V3C/V3M to OBJ or FBX tool.  

Given the direction modern game development tools have gone, there is merit to having access to the original models for modding purposes.  Most texturing tools now are interactive and in 3D space.  Resorting to painting over textures in something like Photoshop is antiquated, slow, inaccurate, and limiting.

A model extractor would go far for the speed and quality of mod-making for this game.
